### ISSUE TITLE
Crash when item list has only one item

### ISSUE THREAD

**Issue by hungdev** [sag1v/react-elastic-carousel#101]
here is code
https://codesandbox.io/s/eager-franklin-oweh0

```
import React from "react";
import "./styles.css";
import Carousel from "react-elastic-carousel";

const Item = ({ someProp }) => {
  return <div className="item">{someProp}</div>;
};
export default function App() {
  return (
    <div className="App">
      <h1>Hello CodeSandbox</h1>
      <h2>Start editing to see some magic happen!</h2>
      <Carousel
        itemsToShow={1}
        onChange={(currentItem, pageIndex) =>
          alert(JSON.stringify({ currentItem, pageIndex }))
        }
      >
        <Item someProp="cool1">1</Item>
      </Carousel>
    </div>
  );
}
```
![image](https://user-images.githubusercontent.com/12009276/98011759-dcfd9c80-1e2a-11eb-88db-81b0929e0b6d.png)

**Comment by sag1v** [comment]
Hi, thanks!
I think there is a bug indeed, and i think i know [where it is](https://github.com/sag1v/react-elastic-carousel/blob/master/src/react-elastic-carousel/components/Carousel.js#L276).

In fact we may have more bugs, I forgot to convert children to an array (`React.Children.toArray`) before i access `.length`.

I will fix this soon, great catch.

**Comment by sag1v** [comment]
fixed as of `v0.9.4`
